pkg/nsm/server/server.go

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2018 Cisco and/or its affiliates.
```

Comment:
This should become a Ligato plugin, per our discussion.

Reply:
Right I will drop this file for now.